doc/modules/preprocessing.rst

```diff
@@ -453,9 +430,11 @@ The normalizer instance can then be used on sample vectors as any transformer::
 
 Note: L2 normalization is also known as spatial sign preprocessing.
 
+See also :func:`normalize`.
+
```

Comment:
Same here, but for `sklearn.preprocessing.Normalizer`.

Reply:
ok